dashboard/app/controllers/api/v1/amazon_future_engineer_controller.rb

```diff
@@ -0,0 +1,13 @@
+require 'cdo/contact_rollups/v2/pardot_helpers'
+
+class Api::V1::AmazonFutureEngineerController < ApplicationController
+  include PardotHelpers
+
+  skip_before_action :verify_authenticity_token
```

Comment:
Just a reminder that we'll want to actually require an authenticity token before we're done with this feature.